src/auth/auth.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Body, Post, HttpCode, UseGuards } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthDto } from './dto';
+import { LocalAuthGuard } from './guards/local-auth.guard';
+import { User } from '@prisma/client';
+import { AuthGuard } from '@nestjs/passport';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
```

Comment:
this can be readonly:

```suggestion
  constructor(private readonly authService: AuthService) {}
```